core/src/banking_stage.rs

```diff
@@ -1166,16 +1267,31 @@ impl BankingStage {
         );
         process_tx_time.stop();
         let unprocessed_tx_count = unprocessed_tx_indexes.len();
+        inc_new_counter_info!(
+            "banking_stage-unprocessed_transactions",
+            unprocessed_tx_count
+        );
```

Comment:
Can you please add this metric and the other one above to `banking_stage_stats` as well?